internal/gcsx/content_type_bucket.go

```diff
@@ -45,6 +46,14 @@ func (b contentTypeBucket) CreateObject(
 	return
 }
 
+func (b contentTypeBucket) CreateAsyncObjectWriter(
+	ctx context.Context,
+	req *gcs.CreateObjectRequest,
+	writeChunkSize int,
+	progressFunc func(int64)) (gcs.ObjectWriter, error) {
+	return nil, fmt.Errorf("not implemented yet")
```

Comment:
This method is not called anywhere in the main flow. why not simply call the wrapped method?

Reply:
Done.

Reply:
I meant for all bucket implementations. But if you want to do it in follow-up PR i am fine. But we will not be test until all implementations are updated to call the wrapped implementation.